test/fixtures/dotenv/valid.env

```diff
@@ -33,3 +33,21 @@ RETAIN_INNER_QUOTES_AS_BACKTICKS=`{"foo": "bar's"}`
 TRIM_SPACE_FROM_UNQUOTED=    some spaced out string
 EMAIL=therealnerdybeast@example.tld
     SPACED_KEY = parsed
+
+MULTI_DOUBLE_QUOTED="THIS
```

Comment:
Can you add one without any quote?

Reply:
Done

Reply:
This is not needed, since we have tests without quotes.

Reply:
@anonrig multiline tests without quotes?

Reply:
Multiline without quotes is not supported.

Reply:
That's why I asked for a test, to guarantee that.